_sass/blocks/_header.scss

```diff
@@ -48,6 +46,47 @@ header,
   width: 100%;
 }
 
+.header-left {
+  @include span-columns(6);
+
+  @include respond-to(medium-up) {
+    @include span-columns(3);
+  }
+}
+
+.header-right {
+  float: right;
+
+  &.header-bars {
+    display: inline-block;
+
+    icon {
+      @include font-size(1.5);
+    }
+  }
+
+  &.header-nav {
+    display: none;
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 72 to appear before line 60. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets